### PR TITLE
Read layer information from mvd3 directly instead of tsv

### PR DESCRIPTION
**Pull request by WeinaJi**

### CODE REVIEW

File: include/mvdtool/bits/mvd3_misc.hpp [hunk 1]
```diff
@@ -18,6 +18,7 @@
  */
 #pragma once
 
+#include "boost/lexical_cast.hpp"
```
Comment (matz-e) [inline include/mvdtool/bits/mvd3_misc.hpp:21]: Since this is not a local include, please use `<>`.

**Review by matz-e — COMMENTED** [review]
Looks great, thank you!

One small nitpick here, otherwise it's good from my side.

**Review by matz-e — APPROVED** [review]
I'll let Tristan sort includes :)

**Review by ferdonline — APPROVED** [review]
LGTM. I would just like to see some description, namely why string instead of int, why we now expect the data from the mvd file instead of tsv. Can we expect implications in that behavior change.
If there is a related ticket please mention it.

**Comment by WeinaJi** [comment]
> LGTM. I would just like to see some description, namely why string instead of int, why we now expect the data from the mvd file instead of tsv. Can we expect implications in that behavior change?
> If there is a related ticket please mention it.

The change is required during the discussion in JIRA ticket https://bbpteam.epfl.ch/project/issues/browse/BBPP82-463 
To summarize, the tsv file will be abandoned and the layer information should be read directly from mvd3, as done for sonata files, which has the common source neuronDB. There is no spec for the data type of layer, purely numbers before, and later mixed with letters (e.g. sub regions). Currently we have MVD3 files with both types,  so string is preferred type to output.

I will add the JIRA number to the commit comment while merging this PR.

**Pull request merged**